### ISSUE TITLE
Remote filesystem not syncing changes in iOS / iPadOS

### ISSUE THREAD

**Issue by lexum0** [godotengine/godot#55839]
### Godot version

3.4

### System information

iPadOS 15

### Issue description

Running the exported iOS project on iPadOS 15 with the launch argument `--remote-fs IP` won't sync any changes made on the editor. Closing and opening the app will load again from the remote filesystem with everything up-to-date, but further changes won't be synced. This affects Scene Changes and Script Changes

### Steps to reproduce

Enable everything in the Debug menu except the Visible options. Run project on iOS device. Make changes to the scene or a script (not sure if you need to save the file but do it) and it won't sync. Relaunch the app on the iOS device and the new changes will be there. Make more changes and they won't sync

### Minimal reproduction project

Make changes like toggle visibility of the root node or open the Label script and comment and uncomment one of the lines to in the _process() function to change the label

[RemoteFS.zip](https://github.com/godotengine/godot/files/7697833/RemoteFS.zip)

**Comment by Calinou** [comment]
Does it work if you use the `--remote-fs` CLI argument on macOS (or any other platform)?

**Comment by lexum0** [comment]
Yes, it works with Windows 10 running the editor and macOS running the `--remote-fs` argument
